node/test/inputtests.ts

```diff
@@ -620,6 +620,18 @@ describe('Input Tests', function () {
 
         done();
     })
+    it('gets delimited input with a Regexp', function (done) {
+        this.timeout(1000);
+
+        var inputValue = 'a,b\nc';
+        process.env['INPUT_DELIM'] = inputValue;
+        im._loadData();
+
+        var outVal = tl.getDelimitedInput(/[,\n]/, ' ', /*required*/false);
```

Comment:
I think this should be `tl.getDelimitedInput('delim', /[,\n]/, /*required*/false);` - that should fix the failing test I believe

Reply:
Looks like you still have an extra input (the `' '` should be removed)